src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -99,8 +99,12 @@ object EventMessages {
     }
 
     override def slack(incomingEvent: ExportStatusEvent): Option[SlackMessage] = {
-      val message = s"The export for the consignment ${incomingEvent.consignmentId} has ${if (incomingEvent.success) "completed" else "failed"}"
-      SlackMessage(List(SlackBlock("section", SlackText("mrkdwn", message)))).some
+      if(incomingEvent.environment != "intg" || !incomingEvent.success) {
```

Comment:
If we add another environment-specific notification like this, we'll probably want to think about abstracting this, but I think it's OK in an if statement for now.